api_specifications/application-api.yaml

```
@@ -0,0 +1,321 @@
+openapi: 3.0.0
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)